vite.config.ts

```diff
@@ -37,6 +38,16 @@ export default defineConfig({
       '~': fileURLToPath(new URL('./node_modules', import.meta.url)),
     },
   },
+  test: {
+    globals: true,
+    exclude: [...configDefaults.exclude, '**/test/e2e', '**/templates/__tests__'],
+    environment: 'jsdom',
+    css: {
+      modules: {
+        classNameStrategy: 'non-scoped'
```

Comment:
See https://vitest.dev/config/#css-modules-classnamestrategy

Note: The warning on Vitest now bypassing CSS module processing